src/App.tsx

```diff
@@ -28,12 +29,12 @@ const router = createBrowserRouter([
     },
 ]);
 
-function App() {
+const App = (): React.ReactElement => {
```

Comment:
What does `React.ReactElement` change compared to how it was before?

Reply:
Explicit type